Blackjack/Program.cs

```diff
@@ -20,7 +20,8 @@ static void Main(string[] args)
             var players = new List<IParticipant> { player, dealer };
             var deck = new Deck();
             var controller = new Controller(input, output, player, dealer, deck);
-            controller.Play();
+            controller.Play(); // TODO: should lines 23 - 24 be a method on Controller?

```

Comment:
no need to combine lines 23 and 24 to a single method in the controller because it's doing two different things. Play is only concerned with producing a game result, whilst display game result is only concerned with outputting the correct messages.